### DISCUSSION TITLE
Chia GUI on second PC does not load my Chia Plots

### DISCUSSION THREAD

**Discussion by cpfish05** [Chia-Network/chia-blockchain#6145] [Support]
i am running Chia GUI on a second Windows PC. i set it up with the same key, it synced and was farming all good. 

but something has happened and the Chia GUI no longer loads the plots i created on my other PC with the same key

ive tried to reinstall, and restart with the 24 key phrase, but the GUI wont load my plots, 

please help

**Comment by cpfish05** [comment]
the ip address changed for the main CA

**Comment by tiny0little** [comment]
post 
```chia keys show``` 
```chia farm summary```
```chia plots show```
from both PCs

**Comment by BrandtH22** [comment]
@cpfish05 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
